### PR TITLE
Set Yarn checksumBehavior to reset

### PR DESCRIPTION
**Pull request by bayasdev**
This should fix checksum mismatches between Windows and macOS/Linux generated lockfiles

See
https://yarnpkg.com/advanced/error-codes#yn0018---cache_checksum_mismatch

### CODE REVIEW

**Comment by harshavardhana** [comment]
https://github.com/minio/console/pull/3364 this PR fixes this already @bayasdev can you verify?

File: .github/workflows/jobs.yaml [hunk 1]
```diff
@@ -19,11 +19,11 @@ concurrency:
...
   lint-job:
     name: Checking Lint
-    runs-on: [ ubuntu-latest ]
+    runs-on: [ubuntu-latest]
     strategy:
       matrix:
-        go-version: [ 1.22.x ]
-        os: [ ubuntu-latest ]
+        go-version: [1.22.x]
+        os: [ubuntu-latest]
```
Comment (harshavardhana) [inline .github/workflows/jobs.yaml:26]: avoid all these unexpected changes these are unrelated to the PR.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: .github/workflows/jobs.yaml (hunk 1 above)
Comment (bayasdev) [reply .github/workflows/jobs.yaml:26]: It was my auto formatter 😅

**Review by harshavardhana — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
